drive/files.py

```diff
@@ -196,15 +203,15 @@ def parents(self):
             self.parents_ids = m["parents"]
 
         parents = []
-        for pid in self.parents_ids:
+        for pid in cast(Iterable[str], self.parents_ids):
```

Comment:
Worth noting: some other type checkers can suss this out automatically (e.g. Pylance). But it seems like MyPy needed this cast. There is similar stuff in one or two other spots.